utility/src/main/java/org/oppia/util/firebase/CrashlyticsWrapper.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Controller for providing custom crash reporting to Firebase Crashlytics */
```

Comment:
Regarding the documentation, this isn't really a controller. Maybe 'logger' would be clearer (per my comment above re: 'CrashLogger').

Reply:
Done